cmd/maya-exporter/app/collector/metrics.go

```diff
@@ -250,6 +252,14 @@ func Metrics(cas string) metrics {
 				Help:      "Total no of replicas connected to cas",
 			},
 		),
+
+		isClientConnected: prometheus.NewGauge(
+			prometheus.GaugeOpts{
+				Namespace: "openebs",
+				Name:      "client_status",
```

Comment:
`client_status` - `iscsi_initiator_login_status`
`iSCSI Initiator to Jiva Target login status: (0, 1): {Not Logged In, Logged In}"